custom/setup.py

```diff
@@ -0,0 +1,53 @@
+import os
+import itertools
+
+from typing import Dict, List
+from setuptools import setup, find_packages
+
+ROOT_PATH = os.path.dirname(__file__)
+PKG_NAME = "mlservermllib"
+PKG_PATH = os.path.join(ROOT_PATH, PKG_NAME)
+
+
+def _load_version() -> str:
+    version = ""
+    version_path = os.path.join(PKG_PATH, "version.py")
+    with open(version_path) as fp:
+        version_module: Dict[str, str] = {}
+        exec(fp.read(), version_module)
+        version = version_module["__version__"]
+
+    return version
+
+
+def _extras() -> Dict[str, List[str]]:
+    extras = {
+        "mllib": ["pyspark==3.0.1"]
+    }
+
+    # Inject key 'all' with all dependencies
+    all_extras = set(itertools.chain(*extras.values()))
+    extras.update({"all": list(all_extras)})
+
+    return extras
+
+
+setup(
+    name=PKG_NAME,
+    version=_load_version(),
+    url=f"https://github.com/seldonio/{PKG_NAME}.git",
+    author="Seldon Technologies Ltd.",
+    author_email="hello@seldon.io",
+    description="ML server",
+    packages=find_packages(),
+    install_requires=[
```

Comment:
May add this line：
python_requires=">=3.6"
Because used f-strings in your code

Reply:
Install mlserver here instead. 
Will fix the python version in the upstream mlserver.